src/App.jsx

```diff
@@ -22,6 +26,11 @@ const App = () => (
       <Route path="/reset_password/:token" exact component={withAlert(ResetPasswordContainer)} />
       <Route path="/article" exact component={withAlert(CreateArticleComponent)} />
       <Route path="/article/:slug" exact component={withAlert(ReadUpdateArticleContainer)} />
+      <Route path="/reset_password/:token" exact component={ResetPasswordContainer} />
```

Comment:
@Ramon3162 You have two `reset_password` routes. Please remove this one. Remain with the one with `withAlert`.

Reply:
I have sorted this. Thank you for that catch.